udata/api/__init__.py

```diff
@@ -239,6 +239,7 @@ def fix_apidoc_throbber():
 
 
 class API(Resource):  # Avoid name collision as resource is a core model
+    @api.hide
```

Comment:
Because if not present every endpoint add an OPTIONS method in the swagger spec (and so in the API doc).
The OPTIONS fix is a quick fix until it's fixed in flask-restplus